client/src/Contexts/Consultation.context.tsx

```diff
@@ -91,7 +96,11 @@ const ConsultationContextProvider = (props: Props) => {
 
   const changePainLevel = (painLevel: number): void => setPainLevel(painLevel);
 
-  const updateDoctorId = (id: number): void => setDoctorId(id);
+  const updateDoctor = (
+    id: number,
+    firstName: string,
+    lastName: string
+  ): void => setDoctor(id);
```

Comment:
On line 23 updateDoctor takes  language and docPublicCode as args as well. Also in the setDoctor on this line is there a reason why we're only passing the id and not everything else?